sass/views/download_sass_data_site.py

```diff
@@ -151,3 +151,52 @@ def download_sass_summary_data(request):
     )
 
     return JsonResponse(get_response(PROCESSING_STATUS, filename))
+
+
+def download_sass_taxon_data(request, **kwargs):
+    """
+    Download all sass data taxon
+    """
+    csv_name = request.GET.get('csvName')
+    location_site = request.GET.get('siteId')
+    site_visit = SiteVisit.objects.filter(location_site=location_site)
```

Comment:
this should be a latest site_visit id, you can add a context_data for site visit id in sass_dashboard.py line 134.